spec/zooniverse_social/posts_spec.rb

```diff
@@ -28,14 +28,22 @@ module ZooniverseSocial
         expect(blog_updater).to have_received :update
         expect(daily_updater).to have_received :update
       end
+
+      it 'calls the daily updater' do
```

Comment:
Yes, my bad I left that in but it is redundant sort of. Anyhow, I don't think we want the test to fail unless `:new` is called, so shall I just remove it?